*** No rule to make target 'lib', needed by 'install'

```
 getcount.o probe.o info.o fort.o mpif.o
mpi.o send.o recv.o collective.o req.o list.o handles.o comm.o error.o ic_merge.o group.o time.o pack.o type.o type_const.o copy.o op.o cart.o getcount.o probe.o info.o fort.o mpif.o
rm -f libmpi-serial.a
ar cruv libmpi-serial.a mpi.o send.o recv.o collective.o req.o list.o handles.o comm.o error.o ic_merge.o group.o time.o pack.o type.o type_const.o copy.o op.o cart.o getcount.o probe.o info.o fort.o mpif.o
a - mpi.o
a - send.o
a - recv.o
a - collective.o
a - req.o
a - list.o
a - handles.o
a - comm.o
a - error.o
a - ic_merge.o
a - group.o
a - time.o
a - pack.o
a - type.o
a - type_const.o
a - copy.o
a - op.o
a - cart.o
a - getcount.o
a - probe.o
a - info.o
a - fort.o
a - mpif.o
ranlib libmpi-serial.a
root@modex:/mpi-serial# make install
make: *** No rule to make target 'lib', needed by 'install'.  Stop.
```

Same error I get when trying to use the MCT version.  https://github.com/MCSclimate/MCT/issues/37

--- a/Makefile
+++ b/Makefile
@@ -8,7 +8,7 @@ VPATH=$(SRCDIR)/mpi-serial
 MODULE		= mpi-serial
 
 SRCS_F90	= fort.F90 \
-                  mpif.F90
+		  mpif.F90
 
 SRCS_C		= mpi.c \
 		  send.c \
@@ -17,20 +17,20 @@ SRCS_C		= mpi.c \
 		  req.c \
 		  list.c \
 		  handles.c \
-                  comm.c \
+		  comm.c \
 		  error.c \
-                  ic_merge.c \
-                  group.c \
-                  time.c \
-                  pack.c \
-                  type.c \
-                  type_const.c \
-                  copy.c \
-                  op.c \
-                  cart.c \
-                  getcount.c \
-                  probe.c \
-                  info.c
+		  ic_merge.c \
+		  group.c \
+		  time.c \
+		  pack.c \
+		  type.c \
+		  type_const.c \
+		  copy.c \
+		  op.c \
+		  cart.c \
+		  getcount.c \
+		  probe.c \
+		  info.c
 
 
 OBJS_ALL	= $(SRCS_C:.c=.o) \
@@ -80,14 +80,11 @@ clean:
 	/bin/rm -f *.o ctest ftest $(LIB) mpi.mod config.log config.status
 	cd tests ; $(MAKE) clean
 
-tests:
+tests: lib$(MODULE).a
 	cd tests; make
 
-install: lib
+install: lib$(MODULE).a
 	$(MKINSTALLDIRS) $(libdir) $(includedir)
 	$(INSTALL) lib$(MODULE).a -m 644 $(libdir)
 	$(INSTALL) mpi.h -m 644 $(includedir)
 	$(INSTALL) mpif.h -m 644 $(includedir)
-
-
-

--- a/mpi.h
+++ b/mpi.h
@@ -251,7 +251,7 @@ typedef int MPI_Info;         /* handle */
  * you can use 'protify.awk' and paste the output here.
  *
  */
-
+extern int MPI_Get_library_version(char *version, int *resultlen);
 
 extern int MPI_Intercomm_create(MPI_Comm local_comm, int local_leader,
                           MPI_Comm peer_comm, int remote_leader,

